examples/MultiTouch/MultiTouch.ino

```diff
@@ -0,0 +1,39 @@
+/*
+  Copyright (c) 2021 ilufang
+  See the readme for credit to other people.
+
+  Multi-touch touchscreen example
+  Draw 7 parallel lines across the screen.
+  Open Microsoft Whiteboard, select a brush and press the button.
+
+  See HID Project documentation for more infos
+  https://github.com/NicoHood/HID/wiki/Gamepad-API
+*/
+
+#include <HID-Project.h>
+
+const int pinButton = 2;
+
+void setup() {
+	pinMode(pinButton, INPUT_PULLUP);
+	MultiTouch.begin();
+}
+
+void loop() {
+	while(digitalRead(pinButton));
+
+	int16_t x = 2000, y = 2000;
+
+	for (; x <= 8000; x+=10) {
+		for (int i = 0; i < 7; i++) {
```

Comment:
"Hybrid Mode" issue below.

2 fingers will fully exercise the report. Anything more than 2 exercises the Hybrid Mode reporting. I went over an issue with reporting not-multiple-of-2 fingers when unused fingers are not zeroed, so I decided to report 7 here. This will have fully populated reports, multiple reports for hybrid mode and partially populated reports.

Reply:
And why do you define 10 fingers then? I think I dont understand that correct.

Reply:
In the current state, 2 is the number of fingers in the report, defined by the descriptor; 10 is the theoretical maximum number of fingers, defined by the feature report:

* Each report has 2 fingers
* If more than 2 fingers is present, we send more reports. Hybrid mode defines how to do it.
* We can send as many reports as desired, but Windows won't process more than 10 (software restriction I guess)
* Windows will say 10-finger support in System Properties
* We don't have a 10-finger report because it will be unnecessarily large for the common case where only 2 fingers are expected. Maybe there are also size limits so we can't pack like 100 touches into a single report, but still able to report them through 20 reports with 5 touches in each.

I'm setting 2/10 because I think that represents the most common scenarios. In fact I'm trying to do something that sends ~8 touches regularly up to max ~36 concurrent touches. I was expecting the user to configure these parameters.